Crashlytics/Crashlytics/Controllers/FIRCLSManagerData.h

```diff
@@ -28,14 +28,14 @@ NS_ASSUME_NONNULL_BEGIN
 @protocol FIRAnalyticsInterop;
 
 /*
- * Manager Data's purpose is to simplify the adding and removing of
+ * FIRCLSManagerData's purpose is to simplify the adding and removing of
```

Comment:
Should an `NS_SWIFT_NAME(CLSManagerData)` be added in Firebase 9?

Reply:
Definitely worth considering for consistency!